src/pages/Heatmap.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { withFauxDOM } from 'react-faux-dom'
+import * as d3 from 'd3';
+import DataSdk from '../SDK/DataSdk';
+import { drawHeatmap } from '../SDK/D3Heatmap.js';
+
+const Heatmap = (props) => {
+    var accessionFromParam = new URLSearchParams(props.location.search).get("accession");
+    const dataSdk = new DataSdk();
```

Comment:
To make my suggestion from above work, we can get the accession from the "searchbox" component on the data page and use that instead. This would allow us to add more error checks and keep things UI-centric. We can also alter the route to the data page after the request for said accession was successful.